packages/blade/package.json

```diff
@@ -249,7 +249,7 @@
     "react-dom": "18.2.0",
     "react-native": "0.72.3",
     "react-native-gesture-handler": "2.9.0",
-    "react-native-reanimated": "3.4.1",
+    "react-native-reanimated": "3.4.2",
```

Comment:
That accordion issue I was talking about was actually an issue in react-native-reanimated which got fixed in this version https://github.com/software-mansion/react-native-reanimated/issues/2768#issuecomment-1691728342